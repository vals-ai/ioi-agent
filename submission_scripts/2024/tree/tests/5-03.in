version https://git-lfs.github.com/spec/v1
oid sha256:f9eb4c3289bd4ef90c363c3b2e109296ab55b722f97a5bd8c268a64d202acb6e
size 3036513
