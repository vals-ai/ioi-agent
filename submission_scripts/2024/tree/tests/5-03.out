version https://git-lfs.github.com/spec/v1
oid sha256:5f9de7d91d32dd6ec71b33b7cfda4c0eea158d23ae7f79ac7d8a02349f0b6b01
size 1137977
